src/commands/ext-configure.ts

```diff
@@ -47,156 +43,72 @@ export default new Command("ext:configure <extensionInstanceId>")
   .action(async (instanceId: string, options: Options) => {
     const projectId = getProjectId(options);
 
-    if (options.local) {
-      if (options.nonInteractive) {
-        throw new FirebaseError(
-          `Command not supported in non-interactive mode, edit ./extensions/${instanceId}.env directly instead`
-        );
-      }
-
-      const config = manifest.loadConfig(options);
-
-      const refOrPath = manifest.getInstanceTarget(instanceId, config);
-      const isLocalSource = isLocalPath(refOrPath);
-
-      let spec: extensionsApi.ExtensionSpec;
-      if (isLocalSource) {
-        const source = await createSourceFromLocation(needProjectId({ projectId }), refOrPath);
-        spec = source.spec;
-      } else {
-        const extensionVersion = await extensionsApi.getExtensionVersion(refOrPath);
-        spec = extensionVersion.spec;
-      }
+    if (options.nonInteractive) {
+      throw new FirebaseError(
+        `Command not supported in non-interactive mode, edit ./extensions/${instanceId}.env directly instead`
```

Comment:
Hmmm, if a user is hitting this error now, they are probably trying to figure out how to do noninteractive deploys. Consider including a manifest docs link to help them figure out the new way to do this?